packages/labs/src/components/tag-input/tagInput.tsx

```diff
@@ -187,7 +188,8 @@ export class TagInput extends AbstractComponent<ITagInputProps, ITagInputState>
         );
     }
 
-    private renderTag = (tag: React.ReactNode, index: number) => {
+    private maybeRenderTag = (tag: React.ReactNode, index: number) => {
+        if (!tag) { return null; }
```

Comment:
been thinking about this... react handles `undefined` and `null` _and `false_` identically, but our code ignores `false`. the `false` case is critical for the `condition && <Element />` pattern--if `condition=false` then the entire expression resolves to `false`.

decided to try out the true react way here and i dig it.